instrumentation/otel-webserver-module/src/nginx/ngx_http_opentelemetry_module.c

```diff
@@ -729,40 +751,56 @@ static void otel_payload_decorator(ngx_http_request_t* r, APPD_SDK_ENV_RECORD* p
 /*
     Stopping an Interaction
 */
-static void otel_stopInteraction(ngx_http_request_t* r, const char* module_name)
+static void otel_stopInteraction(ngx_http_request_t* r, const char* module_name,
+    const char* request_handle_key)
```

Comment:
It's better to keep request_handle_key as void* ig

Reply:
Agreed with void *. But I see anomalies in the code. request handle key inside ngx_http_otel_handles_t is kept as const char*. Hence when we retrieve from ctx, we get const char*. So to keep consistency inside the function, it is kept  as const char*.  Should we change to void* everywhere to be consistent? what are your thoughts?

Reply:
Done